File: Desktop/src/components/messages/CreateGridRequest.js
export default class CreateGridRequest {
    constructor(parentId, columns, rows, columnSpacing, rowSpacing) {
        this.parentId = parentId;
        this.columns = columns;
        this.rows = rows;
        this.columnSpacing = columnSpacing;
        this.rowSpacing = rowSpacing;
        this.action = "CreateGridRequest";
    }
}

File: Mobile/Source/UISleuth/Messages/CreateGridRequest.cs
namespace UISleuth.Messages
{
    internal class CreateGridRequest : Request
    {
        public string ParentId { get; set; }
        public int Columns { get; set; }
        public int ColumnSpacing { get; set; }
        public int Rows { get; set; }
        public int RowSpacing { get; set; }
    }
}